packages/fontforge-of-ocaml/fontforge-of-ocaml.1.1.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "OCaml binding of FontForge"
+description: """
+Library providing OCaml binding of FontForge Python API
+"""
+maintainer: "http://github.com/pbaudin"
+authors: [
+  "Patrick Baudin"
+]
+license: "LGPL-2.1-only"
+homepage: "http://github.com/pbaudin/fontforge-of-ocaml"
+bug-reports: "https://github.com/pbaudin/fontforge-of-ocaml"
+dev-repo: "git+https://github.com/pbaudin/fontforge-of-ocaml"
+tags: ["font" "fontforge" "binding"]
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.09.0"}
+  "base" {>= "v0.12.2"}
+  "fmt" {>= "0.8.9"}
+  "pyml" {>= "20200518"}
+  "ppx_python" {>= "v0.12.0"}
+  "odoc" {with-doc}
+]
+depopts: [
+  "headache"
+]
+conflicts: [
+  "headache" {< "1.05"}
+]
```

Comment:
The semantics of version constraints inside `depopts` is different from the one you are looking for.
The usual semantics that users want is the one provided by adding the constraint in `conflicts`.

In `depopts` it means: "this package optionally depends on this, but only if the version matches the constraint" so it would still be possible for `headache < 1.05` to be installed, it would just not rebuild the package and would break at the next reinstall

Reply:
I see. Thank you for the explanation.